nodejs/eks/cluster.ts

```diff
@@ -939,11 +939,26 @@ export function createCore(
                             // deployment that AWS deployed already as part of cluster creation.
                             const tmpKubeconfig = tmp.fileSync();
                             fs.writeFileSync(tmpKubeconfig.fd, JSON.stringify(kconfig));
+
+                            // Determine if the CoreDNS deployment has a computeType annotation.
+                            const cmdGetAnnos = `kubectl get deployment coredns -n kube-system -o jsonpath='{.spec.template.metadata.annotations}'`;
+                            const getAnnosOutput = childProcess.execSync(cmdGetAnnos, {
+                                env: {
+                                    ...process.env,
+                                    KUBECONFIG: tmpKubeconfig.name,
+                                },
+                            });
+                            const getAnnosOutputStr = getAnnosOutput.toString();
+                            // See if getAnnosOutputStr contains the annotation we're looking for.
+                            if (!getAnnosOutputStr.includes("eks.amazonaws.com/compute-type") ) {
+                                // No need to patch, the annotation is not present.
+                                return;
+                            }
+
                             const patch = [
                                 {
-                                    op: "replace",
+                                    op: "remove",
```

Comment:
fyi: AWS now specifies removing the annotation if present, rather than patching its value to `fargate`.